static/elements/chromedash-timeline.html

```diff
@@ -56,21 +55,40 @@
       '_updateTimeline(selectedBucketId, type, view, useRemoteData)'
     ],
 
+    ready: function() {
+      google.charts.load('current', {'packages': ['corechart']});
+      google.charts.setOnLoadCallback(function() {
+        // If there's an id in the URL, load the property it.
+        var lastSlash = location.pathname.lastIndexOf('/');
+        if (lastSlash > 0) {
+          var id = parseInt(location.pathname.substring(lastSlash + 1));
+          if (String(id) != 'NaN') {
+            this.selectedBucketId = id;
+          }
+        }
+      }.bind(this));
+    },
+
     drawVisualization: function(data, bucketId) {
       var table = new google.visualization.DataTable();
       table.addColumn('date', 'Date');
       table.addColumn('number', 'Percentage');
+      table.addColumn({type:'string', role: 'annotation'});
+      table.addColumn({type:'string', role: 'annotationText'});
 
       var rowArray = [];
       for (var i = 0, item; item = data[i]; ++i) {
         var dateStr = item.date.split('-');
         var date = new Date();
         date.setFullYear(parseInt(dateStr[0]), parseInt(dateStr[1]) - 1, parseInt(dateStr[2]));
-        rowArray.push([
-          date,
-          parseFloat((item.day_percentage * 100).toFixed(6))
-          // parseFloat(item.day_percentage * 100)
-        ]);
+        var row = [date, parseFloat((item.day_percentage * 100).toFixed(6))];
```

Comment:
Is the `toFixed` here left over from the precision truncation stuff?  We don't need that anymore - the full precision exposed by the back-end handler is approved to be made public (see go/chromestatus-usecounter).

Reply:
Yes. Cool, I'll remove all the truncation stuff (backend and frontend parts).

Reply:
Actually, we may still need it for the historical data points. The raw metrics are saved and sent to the client truncated.

Reply:
Ugh. Maybe rely on some helper that does truncation just for data with old dates?

Reply:
Filed https://github.com/GoogleChrome/chromium-dashboard/issues/523 for now to track this. I don't think it hurts to keep the truncation in for now.